src/main/java/com/spotify/confidence/ConfidenceValue.java

```diff
@@ -334,10 +338,22 @@ public com.google.protobuf.Value toProto() {
     }
   }
 
+  public static class StringList extends List {
+    public StringList(java.util.List<String> values) {
+      super(values.stream().map(ConfidenceValue::of).collect(Collectors.toList()));
+    }
+  }
+
+  public static class BooleanList extends List {
+    public BooleanList(java.util.List<Boolean> values) {
+      super(values.stream().map(ConfidenceValue::of).collect(Collectors.toList()));
+    }
+  }
+
```

Comment:
Not super opposed to this as a concept but I also wonder how it would've looked with the generics approach?